tensorflow/python/keras/utils/io_utils.py

```diff
@@ -33,6 +35,38 @@
   h5py = None
 
 
+if sys.version_info >= (3, 6):
+  def _path_to_string(path):
+    if isinstance(path, os.PathLike):
+      return os.fspath(path)
+    return path
+elif sys.version_info >= (3, 4):
+  def _path_to_string(path):
+    import pathlib
+    if isinstance(path, pathlib.Path):
+      return str(path)
+    return path
+else:
+  def _path_to_string(path):
+    return path
+
+
+def path_to_string(path):
+  """
```

Comment:
Please make sure docstring follows the style guide: one short phrase followed by empty line followed by rest of the docstring.